python/ray/dashboard/dashboard.py

```diff
@@ -144,6 +159,17 @@ def get_forbidden(_) -> aiohttp.web.Response:
 
         async def raylet_info(req) -> aiohttp.web.Response:
             D = self.raylet_stats.get_raylet_stats()
+            for address, data in D.items():
+                available_resources = data["availableResources"]
+                total_resources = data["totalResources"]
+                extra_info = ""
+                for resource_name in sorted(available_resources.keys()):
+                    total = total_resources[resource_name]
+                    occupied = total - available_resources[resource_name]
+                    total = format_resource(resource_name, total)
+                    occupied = format_resource(resource_name, occupied)
+                    extra_info += f"{resource_name}: {occupied} / {total}, "
```

Comment:
It's generally preferable to use `", ".join(list_of_strings)` rather than building up a string via `+=` to avoid O(n^2) performance.